flanker/addresslib/address.py

```diff
@@ -73,27 +81,92 @@ def parse(address, addr_spec_only=False, metrics=False):
         None
     """
     mtimes = {'parsing': 0}
-
-    parser = flanker.addresslib.parser._AddressParser(False)
+    lexer = flanker.addresslib.lexer.lexer.clone()
```

Comment:
It's `lexer.lexer` because its the `lexer` global variable in `lexer.py`. I might be able to clean that up by making the lexer/parser into a separate module. The problem with that is that PLY uses a lot of meta-programming to build the lexer and parsing tables so it's not as easy as copy-pasting the code into new files and I didn't think it was worth the effort. I think the import cleanup I did in response to another comment made the lexer usage more readable anyhow.